src/CameraSensor.cc

```diff
@@ -187,7 +187,7 @@ bool CameraSensor::CreateCamera()
   }
 
   // \todo(nkoeng) these parameters via sdf
-  this->dataPtr->camera->SetAntiAliasing(2);
+  this->dataPtr->camera->SetAntiAliasing(cameraSdf->AntiAliasingValue());
```

Comment:
the default value in SDFormat is `4`, so this will represent a behavior change

* https://github.com/gazebosim/sdformat/blob/main/sdf/1.9/camera.sdf#L31

Reply:
hmm it may be worth putting a note in the migration guide about this change, otherwise changes look good to me.

Reply:
6aa4cb9fedfea9c7830bcbb88fcf0b92888b488d